modules/mri_violations/php/NDB_Menu_Filter_Form_mri_violations.class.inc

```diff
@@ -35,97 +35,108 @@ class NDB_Menu_Filter_Form_mri_violations extends NDB_Menu_Filter_Form
+                array('hash' => $hash)
+            );
+
             //if row is found it means that resolve status was previously assigned,
             // check the value and update if needed, else break.
             if (!empty($row)) {
-                foreach($row as $key2 => $val2){
+                foreach ($row as $key2 => $val2) {
                     //found matching
                     if ($key2 == 'Resolved') {
                         // user input matches DB content, so do nothing.
                         if ($val==$val2) {
                             break;
-                        } else { 
-                        // user input doesn't match DB, so we update the DB 
-                            $setArray = array('Resolved'=>(string)$val,'ChangeDate'=>date("Y-m-d H:i:s"));
-                            $whereArray = array('hash'=>$hash);
-                            $DB->update('violations_resolved', $setArray, $whereArray);
+                        } else {
+                            // user input doesn't match DB, so we update the DB
+                            $setArray   = array(
+                                           'Resolved'   => (string)$val,
+                                           'ChangeDate' => date("Y-m-d H:i:s"),
+                                          );
+                            $whereArray = array('hash' => $hash);
+                            $DB->update('violations_resolved', $setArray, $whereArray);// @codingStandardsIgnoreLine
                         }
                     }
                 }
             } else {
-            //if row is not found no resolve status was assigned,
-            // if selection<>0, then insert new row.
+                //if row is not found no resolve status was assigned,
+                // if selection<>0, then insert new row.
                 // no need to insert to DB for Unresolved value.
                 if ($val=='unresolved') {
                     continue;
                 }
-                $newlyResolved = array();
-                $newlyResolved['hash'] = $key;                
-                $newlyResolved['Resolved'] = (string)$val;
-                $newlyResolved['User'] = $user->getUsername();
+                $newlyResolved         = array();
+                $newlyResolved['hash'] = $key;
+                $newlyResolved['Resolved']   = (string)$val;
+                $newlyResolved['User']       = $user->getUsername();
                 $newlyResolved['ChangeDate'] = date("Y-m-d H:i:s");
 
                 // Table 1: mri_protocol_violated_scans
                 $ID_mri_protocol_violated_scans = $DB->pselectOne(
                     "SELECT ID
                     FROM mri_protocol_violated_scans 
-                    WHERE (:hash = md5(concat_WS(':',minc_location,PatientName,SeriesUID,time_run)))",
+                    WHERE (:hash = md5(concat_WS(':',minc_location,PatientName,SeriesUID,time_run)))",// @codingStandardsIgnoreStart
```

Comment:
I think telling PHPCS to ignore coding standards defeats the purpose of using PHPCS